SPECS/apr-util/apr-util.spec

```diff
@@ -1,25 +1,25 @@
-Summary:    The Apache Portable Runtime Utility Library
-Name:       apr-util
-Version:    1.6.1
-Release:        4%{?dist}
-License:    Apache License 2.0
-URL:        https://apr.apache.org/
-Group:      System Environment/Libraries
+Summary:        The Apache Portable Runtime Utility Library
+Name:           apr-util
+Version:        1.6.3
+Release:        1%{?dist}
+License:        Apache License 2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:    http://archive.apache.org/dist/apr/%{name}-%{version}.tar.gz
-%define sha1 %{name}=5bae4ff8f1dad3d7091036d59c1c0b2e76903bf4
+Group:          System Environment/Libraries
+URL:            https://apr.apache.org/
+Source0:        https://archive.apache.org/dist/apr/%{name}-%{version}.tar.gz
+%define     sha1      %{name}=5bae4ff8f1dad3d7091036d59c1c0b2e76903bf4
 %define     apuver    1
```

Comment:
Let's also move macro definition to the top as part of the clean-up. And I know, the linter is annoying with these.;)

Reply:
done